ui/app/services/path-help.js

```diff
@@ -21,14 +24,25 @@ export default Service.extend({
     });
   },
 
-  getNewModel(modelType, backend, owner) {
+  //Makes a call to grab the OpenAPI document.
+  //Returns relevant information from OpenAPI
+  //as determined by the expandOpenApiProps util
+  getProps(helpUrl, path, backend) {
+    return this.ajax(helpUrl, backend).then(help => {
+      let props = help.openapi.paths[path].post.requestBody.content['application/json'].schema.properties;
+      return expandOpenApiProps(props);
+    });
+  },
+
+  getNewModel(modelType, owner, backend) {
     let name = `model:${modelType}`;
     let newModel = owner.factoryFor(name).class;
-    if (newModel.merged || !newModel.prototype.useOpenAPI === true) {
+    if (newModel.merged || newModel.prototype.useOpenAPI !== true) {
       return resolve();
     }
+    let { helpUrl, path } = newModel.prototype.getOpenApiInfo(backend);
```

Comment:
Aha, I was wondering why not computed props, but forgot we were dealing with the model definition here, not an instance - makes sense!